FrontEnd/src/components/CookiesPolicyPage/LinkContainer.jsx

```diff
@@ -0,0 +1,42 @@
+import { NavLink } from 'react-router-dom';
+import styles from './cookiesPolicyComponent.module.css';
+
+const LinkContainer = () => (
+  <div className={styles['link']}>
```

Comment:
Could you align the markup? I would recommend using auto-code formaters like Prettier

Reply:
corrected